modin/__init__.py

```diff
@@ -85,19 +85,19 @@ class Publisher(object):
     def __init__(self, name, value):
         self.name = name
         self.__value = value.title()
-        self.__subs = set()
-        self.__once = collections.defaultdict(set)
+        self.__subs = []
```

Comment:
Do we no longer have unique values? Or do we need just need order?

Reply:
As commented above, we need order, because we need to make sure a callback from `modin.pandas` happens first as it's the one that initializes remote Ray.